package.json

```diff
@@ -1,8 +1,11 @@
 {
   "name": "mobx",
-  "version": "3.1.5",
+  "version": "3.1.6",
   "description": "Simple, scalable state management.",
   "main": "lib/mobx.js",
+  "module": "lib/es2015/mobx.js",
+  "jsnext:main": "lib/es2015/mobx.js",
```

Comment:
Do we really need `jsnext:main`? Rollup certainly prefers `module` instead of `jsnext:main`: https://github.com/rollup/rollup/wiki/jsnext:main

Is there a tool which relies on `jsnext:main` and doesn't work with `module`?